app/templates/components/billing/select-plan.hbs

```diff
@@ -33,7 +33,7 @@
             {{#if (or (and plan.isAnnual this.showAnnual) (and (not plan.isAnnual) (not this.showAnnual)))}}
               <div
                 data-test-plan-box='true'
-                class='billing-plans__box-v2 pointer {{if (eq plan.id this.selectedPlan.id) 'highlight-plan'}}'
+                class='billing-plans__box-v2 pointer {{if (or (eq plan.id this.selectedPlan.id) (eq plan.name "Fixed Price: 5 Jobs")) "highlight-plan"}}'
```

Comment:
@nikolaTrichkovski 
this naming of plans is something that keeps changing.
now it is "Fixed Price: 5 Jobs", previously it was something and in future we don't know what could it be next.
I believe there should be a better way to handle it